crypto/src/main/java/net/consensys/cava/crypto/sodium/Signature.java

```diff
@@ -502,4 +526,63 @@ public static boolean verifyDetached(byte[] message, byte[] signature, Signature
 
     return true;
   }
+
+  /**
+   * Signs a message for a given key.
+   *
+   * @param message The message to sign.
+   * @param secretKey The secret key to sign the message with.
+   * @return The signature prepended to the message
+   */
+  public static Bytes sign(Bytes message, Signature.SecretKey secretKey) {
```

Comment:
No need to fully qualify `SecretKey` here.

Reply:
The other methods that were there already have it fully qualified.  I can change it if you want though.

Reply:
You're right - I'll fix that later.